Delfino/DualCore/SmartWinchDualCore/autonomous_mode.c

```diff
@@ -20,11 +20,22 @@ void waypoint_follower()
         modbus_holding_regs[Target_Y] = waypoints(modbus_holding_regs[Current_Waypoints_Pointer], 1);
         modbus_holding_regs[Target_Z] = waypoints(modbus_holding_regs[Current_Waypoints_Pointer], 2);
 
-        //update target length
-        target_cable_lengths = XYZ_to_length4(  (float) modbus_holding_regs[Target_X], 
-                                                (float) modbus_holding_regs[Target_Y], 
-                                                (float) modbus_holding_regs[Target_Z], 
-                                                (float) modbus_holding_regs[Field_Length]);
+        //update target lengths
++        //Finds current coordinates and from that, relative uplift
++        cur_target_point = tenandsag2coord( (float) modbus_holding_regs[Current_Force_Winch0]*0.0098066500286389f,
```

Comment:
valid check?

Reply:
The valid bit is there, Currently if there is an error in the sag calculation it returns Pythagorean maths instead automatically. I haven't implemented it as didn't know how you wanted that doing

Reply:
looks good here, should i merge it?

idk what to do if the flag is invalid either, im thinking maybe:
-ignore the math and dont move the tether
-or, revert to static-math

what is your opinion?